src/index.js

```diff
@@ -28,7 +28,7 @@ function Multiaddr (addr) {
   }
 
   // default
-  addr = addr || ''
+  addr = addr == null ? '' : addr
```

Comment:
What about:

    if (addr == null) {
      addr = ''
    }

That seems easier to read for future generations ;)

Reply:
It took my a while to figure out that `== null` also matches `undefined`, I'm just so used to `===`. I personally would prefer a `if (addr === undefined || addr === null)`. But that's surely a matter of style/taste, so feel free to leave it with `== null`.